tests/sentry/issues/test_producer.py

```diff
@@ -68,18 +69,18 @@ def test_with_only_occurrence(self) -> None:
 @apply_feature_flag_on_cls("organizations:issue-platform-api-crons-sd")
 class TestProduceOccurrenceForStatusChange(TestCase, OccurrenceTestMixin):
     def setUp(self):
+        self.fingerprint = ["group-1"]
         self.event = self.store_event(
             data={
                 "event_id": "a" * 32,
                 "message": "oh no",
                 "timestamp": iso_format(datetime.now()),
-                "fingerprint": ["group-1"],
+                "fingerprint": self.fingerprint,
             },
             project_id=self.project.id,
         )
         self.group = self.event.group
         assert self.group
-        self.group_hash = GroupHash.objects.filter(group=self.group, project=self.project).first()
```

Comment:
Why'd we stop using `GroupHash` here?

Reply:
We call `process_occurrence_data` on the fingerprint, which is how the callers would use this API. It also lines up better with the error message if we don't find a grouphash